src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFromFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeToFile(report, toFileName);
+    }
+
+    private void writeToFile(String reportString, String toFileName) {
+        try {
+            PrintWriter writer = new PrintWriter(toFileName);
+            writer.print("");
+            writer.close();
+
+            new File(toFileName).createNewFile();
```

Comment:
`new PrintWriter(toFileName).close();` will create file if not exists and clear its content if it exists, that's enough

Reply:
did not work, gave me an error

Reply:
weird, what is the error?